### DISCUSSION TITLE
How to specify which binaries are lambdas?

### DISCUSSION THREAD

**Discussion by ilia-tolliu** [cargo-lambda/cargo-lambda#718] [Q&A]
Whether I run `cargo lambda` in a standalone project or in a workspace, it scans all the binaries and lists those as available lambdas.

How can I configure metadata to either whitelist or ignore specific binaries, so that `cargo lambda` would identify only relevant ones?

Thank you!

**Comment by calavera** [comment]
You have to build them independently with the `--bin` flag, like you'd do with any other Cargo project:

```
cargo lambda build --release --bin THE_LAMBDA_BINARY
```
